cfme/middleware/provider/hawkular.py

```diff
@@ -93,13 +93,13 @@ def num_datasource(self):
     @num_datasource.variant('ui')
     def num_datasource_ui(self, reload_data=True):
         if reload_data:
-            self.summary.reload()
+            self.load_details()
```

Comment:
I guess you understood my recommendation wrong, summary reload is necessary if you want to refresh the values in summary, which you **want** to do in this method ...